tests/scripts/create_coverage_report.py

```diff
@@ -14,40 +13,35 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def merge_all_info_files(start_path, merge_file_name):
-    lcov_list = list()
-    lcov_list.append("lcov")
-
-    root = pathlib.Path(start_path)
-
-    for file_path in root.glob("**/*.info"):
-        file = str(file_path)
-        LOGGER.debug(f"Found info file '{file}'")
-        lcov_list.append("-a")
-        lcov_list.append(file)
-
-    lcov_list.append("-o")
-    lcov_list.append(merge_file_name)
-
-    command.Command(args=" ".join(lcov_list)).run()
-
-
 def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
     path_to_info_files = os.environ["TMT_TREE"].split('/tree')[0] + "/execute/data/"
     path_to_tests_results = os.environ["TMT_TREE"].split('/tree')[0] + "/report/default-0"
     merge_file_name = "merged.info"
-    merge_file_full_path = path_to_tests_results + "/" + merge_file_name
     merge_dir = "/tmp"
     report_dir_name = "/report"
 
-    merge_all_info_files(path_to_info_files, merge_file_full_path)
+    # Copy info files to the ctrl container and run lcov command
+    root = pathlib.Path(path_to_info_files)
+    lcov_list = list()
+    lcov_list.append("lcov")
+    for file_path in root.glob("**/*.info"):
+        file = str(file_path)
+        file_name = file.split("/")[-1]
```

Comment:
file name can be accessed directly from `Path` object using `name` property, for example `file_path.name`